server/src/test/java/io/crate/types/UncheckedObjectTypeTest.java

```diff
@@ -33,36 +33,33 @@
 
 public class UncheckedObjectTypeTest extends ESTestCase {
 
+    UncheckedObjectType uncheckedObjectType = UncheckedObjectType.INSTANCE;
```

Comment:
super minor: Seems uncessary to declare it here (could be `static final` as well), and use directly `INSTANCE` in the test methods?